check_package_version.go

```diff
@@ -0,0 +1,34 @@
+package opslevel
+
+type PackageVersionCheckFragment struct {
+	MissingPackageResult       *CheckResultStatusEnum `graphql:"missingPackageResult"`       // The check result if the package isn't being used by a service.
+	PackageConstraint          *PackageConstraintEnum `graphql:"packageConstraint"`          // The package constraint the service is to be checked for.
```

Comment:
Should `PackageConstraint` be a pointer? Looks like it's non-nullable in the docs